fix: blog card image size and spacing

VedP20 opened the pull request:
What does this PR do?

fix: blog card image size and spacing also new MacBook image is added, also proper spacing between the title and subtitle is added

Fixes #162

Type of change

- New feature (non-breaking change which adds functionality)

Mandatory Tasks

- [X] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

VedP20 commented:
Please do not merge this PR now

VedP20 commented:
need to tweak the image height a bit more because we have images with different sizes and we are using same portfolio component for blogs , courses etc.

VedP20 commented:
you can push it now, let me know if any changes

TheMohit2003 commented on components/data/gears.js:
```diff
@@ -14,7 +14,7 @@ const gears = [
     subtitle:
       "2022 Apple MacBook Pro Laptop with M1 chip: 34.46 cm (13.6-inch) Liquid Retina Display, 8GB RAM, 512GB SSD Storage, Backlit Keyboard, 1080p FaceTime HD Camera. Works with iPhone/iPad; Space Grey",
-    img: "/images/mac.webp",
+    img: "/images/macbook.jpg",
```
There was mac.webp already as a file for MacBook, why did you import a new file mackbook.jpg ?

VedP20 commented:
I changed the image because previous image has different size and while
using it it was expanding out of the card that's why i changed it

On Mon, 12 Jun, 2023, 8:30 am Mohit Pardeshi, ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In components/data/gears.js
> <https://github.com/piyushgarg-dev/piyushgargdev-nextjs/pull/167#discussion_r1226038536>
> :
>
> > @@ -14,7 +14,7 @@ const gears = [
>      title: "MacBook Pro M1 Chip",
>      subtitle:
>        "2022 Apple MacBook Pro Laptop with M1 chip: 34.46 cm (13.6-inch) Liquid Retina Display, 8GB RAM, 512GB SSD Storage, Backlit Keyboard, 1080p FaceTime HD Camera. Works with iPhone/iPad; Space Grey",
> -    img: "/images/mac.webp",
> +    img: "/images/macbook.jpg",
>
> [image: image]
> <https://user-images.githubusercontent.com/99909551/245002589-63fe4a17-dade-4f92-949c-ef65330903fa.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/piyushgarg-dev/piyushgargdev-nextjs/pull/167#discussion_r1226038536>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6AFXN5S6EF3XJCFCBJYAQDXK2A4DANCNFSM6AAAAAAZCN7ZKU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>

piyushgarg-dev commented:
Hey @VedP20 
Thanks for contributing 🙌
The MacBook image that you used is the old intel based 2017 mac air, but titles says Macbook M1 pro, so I would suggest you to go with earlier image or new image of Macbook Pro m1.

VedP20 commented:
Sure, I was looking for non copy right images and found this one that's why I used this one. If you can replace it with an appropriate one that would be helpful.
Thank you

VedP20 commented:
I will replace it with this one ![Adobe_Express_20230612_1007260_1.png](https://github.com/piyushgarg-dev/piyushgargdev-nextjs/assets/125852599/db142543-1509-4f14-8094-41e4ef63ef90)

piyushgarg-dev commented on public/images/macbook.jpg:
@VedP20 Please delete this image 🙌🏻

VedP20 replied:
sure

piyushgarg-dev commented:
@VedP20 I would prefer if we can make some changes to the earlier `mac.webp` itself using CSS, it has a transparent background 👀

VedP20 commented:
okay, I will look into that

VedP20 commented:
Hi, @piyushgarg-dev @TheMohit2003 kindly check this PR , I have replaced the two images to fix the image height issue while keeping the static width and height , also fix the warnings related to images

VedP20 commented:
Can you please review this PR?
